bigtable-dataflow-parent/bigtable-beam-import/pom.xml

```diff
@@ -28,6 +28,9 @@ limitations under the License.
     <mainClass>com.google.cloud.bigtable.beam.Main</mainClass>
     <beam-import-netty.version>4.1.48.Final</beam-import-netty.version>
     <beam-import-google-http-client.version>1.34.1</beam-import-google-http-client.version>
+    <!-- Needed to enable integration tests before we figure out -->
+    <!-- possible misconfiguration of Failsafe plugin -->
+    <skipITs>false</skipITs>
```

Comment:
where is this set to true?

Reply:
I don't believe we have set it explicitly but it is current set to true in effective pom, if we run the following in master: 

```
$ mvn org.apache.maven.plugins:maven-help-plugin:3.2.0:effective-pom|grep skipITs
    <skipITs>true</skipITs>
```

I suspect this is caused by some misconfiguration of Failsafe but does not have enough expertise to debug at this moment.

Reply:
Until we figure out whats going on, can we move it to the profile?